docker/development/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN mkdir /iam
 WORKDIR /iam
 
 COPY package.json /iam
-COPY yarn.lock /iam
 
-RUN yarn install
+RUN yarn install --no-lockfile --silent
```

Comment:
The `yarn.lock` file on the root directory is meant for the node_modules installed on my macing which may differ with what our docker environment needs. So I removed the command to copy `yarn.lock` and put the `--no-lockfile` flash so that it does not check out create a lock file when installing the dependencies in our docker image.
For the `--silent` flag, it suppresses yarn's output during installing. It is not important.